### PR TITLE
Updated disconnected registry and oc-mirror docs

### PR DESCRIPTION
**Pull request by mvazquezc**

### CODE REVIEW

File: mirror-registry/README.md [hunk 1]
```diff
@@ -1,106 +1,111 @@
...
 
 4. Profit. One can validate the installation with the following command:
 
-```
-podman login -u init -p ${REGISTRY_PASSWORD} ${REGISTRY_NAME}:8443
-```
+    > :warning: If you don't have the self-signed certificate in your trust store you may need to add `--tls-verify=false` to the podman login command below.
+
+    ~~~sh
+    podman login -u admin -p n0tTh4tS3cur3 ${REGISTRY_NAME}:8443
```
Comment (karmab) [inline mirror-registry/README.md]: better to define a REGISTRY_PASSWORD variable to ease copy/pasting

File: mirror-registry/README.md [hunk 2]
```diff
@@ -1,106 +1,111 @@
...
-- It was not possible to specify a different user than init in the first release. We opened [https://github.com/quay/mirror-registry/pull/46](https://github.com/quay/mirror-registry/pull/46) to address it, which got merged so this will be possible in an upcoming release.
-- IPV6 isn't currently supported by quay. We opened [https://github.com/quay/mirror-registry/pull/47](https://github.com/quay/mirror-registry/pull/47) to address it by using a side haproxy container, but this got closed. For the time beeing, the folder [ipv6](ipv6) contains assets allowing to deploy an extra container in an ipv6 context. This can be deployed with the following instructions:
+* The command can also target a remote host, although we didn't see too much benefit to going with such approach.
+* IPv6 isn't currently supported by quay. We opened [https://github.com/quay/mirror-registry/pull/47](https://github.com/quay/mirror-registry/pull/47) to address it by using a side HAProxy container, but this got closed. For the time being, the folder [IPv6](ipv6) contains assets allowing to deploy an extra container in an IPv6 context. This can be deployed with the following instructions:
```
Comment (karmab) [inline mirror-registry/README.md]: since the PR got closed we can avoid referencing it

File: mirror-registry/README.md [hunk 3]
```diff
@@ -114,24 +119,25 @@ apiVersion: mirror.openshift.io/v1alpha2
...
+    channels: # OCP Versions to mirror
+      - name: stable-4.10
+        minVersion: 4.10.10
+        maxVersion: 4.10.20
+    graph: true
   operators: # Operators we want to mirror
-    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.9 # References entire catalog
-      headsOnly: false # References latest version of each operator in catalog (true is the default value and can be omitted)
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.10
+      full: true
```
Comment (karmab) [inline mirror-registry/README.md]: do we want full though?

File: mirror-registry/README.md [hunk 4]
```diff
@@ -114,24 +119,25 @@ apiVersion: mirror.openshift.io/v1alpha2
...
       packages:
         - name: local-storage-operator
-          channels:
-            - name: 'stable'
         - name: advanced-cluster-management
-          channels:
-            - name: "release-2.4"
+        - name: openshift-gitops-operator
+        - name: ocs-operator
+        - name: odf-operator
```
Comment (karmab) [inline mirror-registry/README.md]: i dont think you need both ocs-operator and odf-operator

File: mirror-registry/README.md [hunk 5]
```diff
@@ -1,106 +1,111 @@
...
-- SSL_KEY is the path of the corresponding ssl key
+        > :information_source: The command below will initialize the registry with the user `admin` and the password `n0tTh4tS3cur3`.
 
-```
-SSH_KEY="/root/.ssh/id_rsa"
-/usr/bin/mirror-registry install --quayHostname ${REGISTRY_NAME} --sslCert ${SSL_CERT} --sslKey ${SSL_KEY} --initPassword ${REGISTRY_PASSWORD} --ssh-key ${SSH_KEY}
-```
+        ~~~sh
+        SSH_KEY="${HOME}/.ssh/id_rsa"
+        /usr/bin/mirror-registry install --quayHostname ${REGISTRY_NAME} --sslCert /etc/quay-install/certs/registry-cert.pem --sslKey /etc/quay-install/certs/registry-key.pem --initUser admin --initPassword n0tTh4tS3cur3 --ssh-key ${SSH_KEY}
```
Comment (karmab) [inline mirror-registry/README.md]: i think at this point, we can specify a username too?

File: mirror-registry/README.md (hunk 1 above)
Comment (mvazquezc) [reply mirror-registry/README.md]: Fixed, added  REGISTRY_USER and REGISTRY_PASSWORD

File: mirror-registry/README.md (hunk 2 above)
Comment (mvazquezc) [reply mirror-registry/README.md]: Done.

File: mirror-registry/README.md (hunk 4 above)
Comment (mvazquezc) [reply mirror-registry/README.md]: Removed odf.

File: mirror-registry/README.md (hunk 5 above)
Comment (mvazquezc) [reply mirror-registry/README.md]: It was there `--initUser`.

File: mirror-registry/README.md (hunk 3 above)
Comment (mvazquezc) [reply mirror-registry/README.md]: Removed.

File: mirror-registry/README.md (hunk 2 above)
Comment (karmab) [reply mirror-registry/README.md]: i meant both PRS

File: mirror-registry/README.md (hunk 3 above)
Comment (karmab) [reply mirror-registry/README.md]: can it be full: false instead

**Review by karmab — APPROVED** [review]

**Pull request merged**